pkgs/by-name/zo/zotify/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  fetchFromGitHub,
+  python3Packages,
+  protobuf_21,
+  lib,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "zotify";
+  version = "0.16.13";
+
+  src = fetchFromGitHub {
+    owner = "zotify-dev";
+    repo = "zotify";
+    rev = "5da27d32a1f522e80a3129c61f939b1934a0824a";
+    hash = "sha256-KA+Q4sk+riaFTybRQ3aO5lgPg4ECZE6G+By+x2uP/VM=";
+  };
+
+  patchPhase = ''
+    substituteInPlace ./setup.cfg --replace-fail "protobuf==3.20.1" "protobuf"
+  '';
+
+  pyproject = true;
+
+  nativeBuildInputs = [ python3Packages.setuptools ];
```

Comment:
This should be `build-systems`, there have been some relatively recent changes surrounding the python wrapper 

Read around here
https://nixos.org/manual/nixpkgs/unstable/#buildpythonpackage-parameters